libraries/chain/controller.cpp

```diff
@@ -1518,6 +1523,8 @@ struct controller_impl {
       auto& bb = pending->_block_stage.get<building_block>();
       const auto& pbhs = bb._pending_block_header_state;
 
+      const auto& gpo = self.get_global_properties();
+
```

Comment:
Is there a reason for moving this declaration earlier?  The only new use is still in scope from the old location.

Reply:
No idea why, fixed.